Web/SFB.Web.UI/Helpers/HistoricalChartBuilder.cs

```diff
@@ -2051,6 +2051,7 @@ <li>refuse collection
                 },
                 new ChartViewModel()
                 {
+                    Id = 55,
                     Name = "Non-classroom support staff - excluding auxiliary staff (Full Time Equivalent)",
```

Comment:
Shouldn't these be, at the least, constants rather than magic strings in the middle of the code?

Reply:
Most definitely. No idea why it was done this way, but I noticed this ID was missing so added it in.